components/Tabs/src/Theming/MDCTabs+MaterialTheming.m

```diff
@@ -0,0 +1,79 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "MDCTabs+MaterialTheming.h"
+
+#import <Foundation/Foundation.h>
+
+#import <MaterialComponents/MaterialColorScheme.h>
+#import <MaterialComponents/MaterialTypographyScheme.h>
+
+static const CGFloat kUnselectedTitleOpacity = (CGFloat)0.6;
+static const CGFloat kUnselectedImageOpacity = (CGFloat)0.6;
```

Comment:
Tip: Name these `kUnselectedOpacity` rather than having separate values.

Reply:
Addressed in 7f1c93e58fc931a53c7cc7bfd5debf7b157bd6aa